x-pack/legacy/plugins/apm/public/components/app/TransactionDetails/WaterfallWithSummmary/WaterfallContainer/Waterfall/waterfall_helpers/waterfall_helpers.ts

```diff
@@ -244,7 +249,8 @@ export function getWaterfall(
       items: [],
       errorsPerTransaction,
       errorsCount: sum(Object.values(errorsPerTransaction)),
-      serviceColors: {}
+      serviceColors: {},
```

Comment:
I'm wondering if `entryTransactionId ` is still necessary here, you could maybe get it from `trace.errorDocs`?
```js
export function getWaterfall(
  { trace, errorsPerTransaction }: TraceAPIResponse,	  { trace, errorsPerTransaction }: TraceAPIResponse,
  entryTransactionId?: Transaction['transaction']['id']	  entryTransactionId?: Transaction['transaction']['id']
): IWaterfall {	): IWaterfall {
```

Reply:
not sure about omitting `entryTransactionId` since it comes from the URL params